### ISSUE TITLE
womp womp

### ISSUE THREAD

**Issue by mochji** [twitter/the-algorithm#1939]
**Describe the bug**
elon musk forces me to scream i love tw- I MEAN X WAIT NO PLEASE I MEANT X!

**To Reproduce**
1. Go to 'x.com'

**Expected behavior**
a picture of a garbage can appears

**Environment**
Firefox 120.0.1 64 bit, Arch Linux 6.6.6 x86_64

**Comment by tactipus** [comment]
lol

**Comment by ghost** [comment]
Will be fixed in firefox 133.7

**Comment by tactipus** [comment]
i use netscape, though :/

**Comment by ghost** [comment]
ok cool

**Comment by tactipus** [comment]
you're bad at improv, I see

**Comment by mochji** [comment]
yeah

**Comment by mochji** [comment]
> you're bad at improv, I see

yeah

**Comment by ghost** [comment]
> you're bad at improv, I see

cool

**Comment by Tintedfireglass** [comment]
> **Describe the bug**
> elon musk forces me to scream i love tw- I MEAN X WAIT NO PLEASE I MEANT X!
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 1. Go to 'x.com'
> 
> **Expected behavior**
> a picture of a garbage can appears
> 
> **Environment**
> Firefox 120.0.1 64 bit, Arch Linux 6.6.6 x86_64

Exactly! I don't see any garbage! Who's the raccoon over there at X. They want to keep their dumpster only to themselves 😡💢

**Comment by Winterwolf365** [comment]
i like this

**Comment by StormLight14** [comment]
WOMP WOMPPP

**Comment by catsmells** [comment]
go on catterchan if you don't like it

**Comment by mppbtw** [comment]
they fixed this bug yet? its getting annoying at this point

**Comment by mochji** [comment]
no its gotten worse, theres this weirdo called "elon" saying "my tummy hurts"

**Comment by ghost** [comment]
maybe if this Elon fellow stopped eating all the trash himself instead of sharing it his tummy wouldn't hurt

**Comment by tiagozip** [comment]
>**Expected behavior**
> a picture of a garbage can appears

bro's roasting elon musk
